media-libs/libgfx/libgfx-1.1.0.ebuild

```diff
@@ -1,10 +1,11 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
```

Comment:
If we bump the EAPI of an ebuild we should revision bump it as well to trigger a rebuild for users who have this package installed.